varats/varats/tools/driver_casestudy.py

```diff
@@ -62,7 +62,7 @@ def main() -> None:
     elif args['subcommand'] == 'view':
         __casestudy_view(args)
     elif args['subcommand'] == 'cleanup':
-        __casestudy_cleanup(args)
+        __casestudy_cleanup(args,parser)
```

Comment:
```suggestion
        __casestudy_cleanup(args, parser)
```
Just formatting thins but running yapf once should automatically fix everything :smile: